styleguide/source/_patterns/03-templates/04-topic/_topic.scss

```diff
@@ -11,25 +27,55 @@
   }
 
   // First column.
-  &:nth-child(1n) {
-    order: 2;
+  &:nth-child(1) {
+    grid-area: 3 / 1 / 3 / 1;
+
+    @include breakpoint($bp-small) {
+      grid-area: 1 / 2 / 1 / 2;
+    }
+
     @include breakpoint($bp-med) {
-      order: 1;
+      grid-area: 1 / 1 / 4 / 1;
+      padding-left: 14px;
     }
   }
 
   // Center column
-  &:nth-child(2n) {
-    order: 1;
+  &:nth-child(2) {
+    grid-area: 1 / 1 / 1 / 1;
+
+    @include breakpoint($bp-small) {
+      grid-area: 1 / 1 / 4 / 1;
+    }
+
+    @include breakpoint($bp-med) {
+      grid-area: 1 / 2 / 4 / 2;
+    }
+  }
+
+  &:nth-child(3) {
+    grid-area: 2 / 1 / 2 / 1;
+
+    @include breakpoint($bp-small) {
+      grid-area: 2 / 2 / 2 / 2;
+    }
+
```

Comment:
Is this grid replacement work directly related to the wayfinder?  If not, please document in the technical implementation notes on the PR.